doc-content/kogito-docs/src/main/asciidoc/creating-running/con_kogito-automation.adoc

```diff
@@ -31,11 +31,11 @@ For example, {PRODUCT} is compatible with the following technologies:
 * *GraalVM* enables you to use native compilation with {PRODUCT}, resulting in fast start-up times and minimal footprint. For example, a native {PRODUCT} service starts in about 0.003ms, about 100 times faster than a non-native start-up. Fast start-up is almost a necessity in a cloud ecosystem, especially if you need small serverless applications.
 * *Knative* enables you to build serverless applications with {PRODUCT} that you can scale up or down (to zero) as needed.
 * *Prometheus* and *Grafana* are compatible with {PRODUCT} services for monitoring and analytics with optional extensions.
-* *Kafka*, *Infinispan*, and *Keycloak* are also some of the middleware technologies that {PRODUCT} uses internally.
+* *Kafka*, *Infinispan*, and *Keycloak* are also some of the middleware technologies that {PRODUCT} supports.
 
 == Domain-specific flexibility
 
-{PRODUCT} adapts to your business domain instead of forcing you to modify your domain to work with {PRODUCT}. You can expose your {PRODUCT} services with domain-specific APIs, based on the processes and rules that you have defined.
+{PRODUCT} adapts to your business domain instead of forcing you to modify your domain to work with {PRODUCT}. You can expose your {PRODUCT} services with domain-specific APIs that are based on the processes and rules that you have defined and that do not include third-party or internal APIs.
```

Comment:
not sure what it does mean.

Reply:
@spolti, Ricardo suggested to clarify that the APIs, which are generated by processes, are not third party or internal. So added that. If you have some other way to clarify that or think it unnecessary, please suggest alternative.

Reply:
I find this sentence extremely confusing as well. It's also a relatively long sentence. If you want to add the "do not include third party or internal APIs", I would make it a separate sentence ... But I'm tempted to say that I would remove that entire piece .... (it doesn't really add much value IMO).

Reply:
Duncan, ack.

@DuncanDoyle, @spolti, @ricardozanini, I've revised as follows. Work for you guys? Ricardo?

>You can expose your {PRODUCT} services with domain-specific APIs, based on the processes and rules that you have defined. Domain-specific APIs for {PRODUCT} services do not require third-party or internal APIs.

Reply:
looks better now.